presto/eval.py

```diff
@@ -35,15 +35,14 @@
 class Hyperparams:
     lr: float = 2e-5
     max_epochs: int = 100
-    batch_size: int = 64
-    patience: int = 10
-    num_workers: int = 4
+    batch_size: int = 2048
+    patience: int = 3
+    num_workers: int = 8
 
 
 class WorldCerealEval:
-    name = "WorldCerealCropland"
```

Comment:
We probably want to keep the Cropland code too, right? Or is it being completely replaced with the crop type code?

Reply:
addressed it here https://github.com/WorldCereal/presto-worldcereal/commit/a350280be17c7715d4022ea78cd9f478e61bfa3c